packages/typespec-azure-resource-manager/lib/decorators.tsp

```diff
@@ -182,6 +182,12 @@ extern dec armCommonTypesVersion(
  */
 extern dec armVirtualResource(target: Model);
 
+/**
+ * This decorator is used on resources that do not satisfy the definition of a resource
+ * but need to be identified as such.
+ */
+extern dec customAzureResource(target: Model);
```

Comment:
I think we want this in the namespace `Azure.ResourceManager.Legacy`

Reply:
moved